pkg/storaged/utils.js

```diff
@@ -617,6 +617,30 @@ function get_children(client, path) {
     return children;
 }
 
+export function get_all_children(client, path) {
```

Comment:
Excuse my ignorance here -- does this need to consider partitions on device block objects as well? E.g. "sda1" is also a child of "sda", no?

Reply:
I see, thanks for the doc comment!